README.md

```diff
@@ -195,7 +197,7 @@ In those cases, you might try following these instructions to optimize the memor
 
 ### Mbed TLS configuration
 
-The custom Mbed TLS configuration can be set by adding `"macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_config.h\""]` to the `.json` file. The [example Mbed TLS config](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mbedtls_config.h) minimizes the RAM and ROM usage of the application. The configuration works on K64F, but it is not guaranteed to work on every Mbed Enabled platform.
+The custom Mbed TLS configuration can be set by adding `"macros": ["MBEDTLS_USER_CONFIG_FILE=\"mbedtls_config.h\""]` to the `mbed_app.json` file. The [example Mbed TLS config](https://github.com/ARMmbed/mbed-os-example-mesh-minimal/blob/master/mbedtls_config.h) minimizes the RAM and ROM usage of the application. The configuration works on K64F, but it is not guaranteed to work on every Mbed Enabled platform.
```

Comment:
Probably shouldn't reference the full path here especially as in releases this will be wrong (ie master is not released). Just the filename should be sufficient

Reply:
Thanks, good point, fixed.

Reply:
@adbridge , is there anything else that you would like me to modify or can the review be approved now?